src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilterTest.java

```diff
@@ -472,12 +518,17 @@ public void testTagsAreClearedEachRun() {
      *
      * @return {@code Tag} list
      */
-    private static List<?> getTagsAfterExecution(SuppressionCommentFilter filter,
+    private static List<Comparable<Object>> getTagsAfterExecution(SuppressionCommentFilter filter,
             String filename, String... lines) {
         final FileContents contents = new FileContents(filename, lines);
-        contents.reportSingleLineComment(1, 0);
+        for (int lineNo = 1; lineNo <= lines.length; lineNo++) {
+            final int colNo = lines[lineNo - 1].indexOf("//");
+            if (colNo >= 0) {
+                contents.reportSingleLineComment(lineNo, colNo);
+            }
+        }
         final TreeWalkerAuditEvent dummyEvent = new TreeWalkerAuditEvent(contents, filename,
-                new LocalizedMessage(1, null, null, null, null, Object.class, null), null);
+                new LocalizedMessage(1, null, null, null, null, Object.class, ""), null);
```

Comment:
changing the message from `null` to `""` here avoids a NPE when passing `event.getMessage()` to the matcher - alternatively, would it be better for that code to check for a null message itself?

Reply:
In the real world, the `key` parameter is always specified, so `customMessage` is optional. But in testing the `key` parameter sometimes is omitted, causing the NPE if the `customMessage` is missing too. I like this change.